src/App.js

```diff
@@ -7,15 +7,23 @@ import EnterName from "./name/EnterName";
 
 export const App = ({ config: { state, indicators } }) => {
   const [petName, setPetName] = useState(state.name);
-  if (petName) {
+  if (indicators.age.maxAge > 20) {
     return (
-      <div className={style.field}>
-        <PetName petName={petName} />
-        <MainField indicators={indicators} state={state} />
+      <div>
+        <p className={style.error}>ERROR</p>
```

Comment:
Great. It would be even better to provide a helpful message explaining to the developer what exactly variable is missing!

Reply:
done